DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -114,7 +114,7 @@
         <a class="nhsuk-button" role="button">
           Edit details
         </a>
-        <a class="nhsuk-button nhsuk-button--secondary  nhsuk-u-margin-left-2" role="button">
+        <a class="nhsuk-button nhsuk-button--secondary" id="change-password-button" role="button">

```

Comment:
I think a simpler way of achieving this would be to put nhsuk-u-margin-right-2 on the `Edit Details` button instead - then you avoid the media query, which could cause issues for large mobiles where the buttons are still on the same line I guess?